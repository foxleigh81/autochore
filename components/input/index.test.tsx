/* global it expect */

import React from 'react';
import { render } from '@testing-library/react';

// Import component files
import Input from './index';

// Tests
describe('InputText component', () => {
  it('renders to match snapshot', () => {
    const { baseElement } = render(<Input name="test" type="text" />);
    expect(baseElement).toMatchSnapshot();
  });
});
