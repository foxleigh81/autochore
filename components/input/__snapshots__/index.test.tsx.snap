// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputText component renders to match snapshot 1`] = `
<body>
  <div>
    <input
      class="input type-text"
      id="test"
      name="test"
      type="text"
      value=""
    />
  </div>
</body>
`;
